extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -24,8 +24,8 @@
 import io.deephaven.parquet.table.metadata.ColumnTypeInfo;
 import io.deephaven.parquet.table.metadata.GroupingColumnInfo;
 import io.deephaven.parquet.table.metadata.TableInfo;
+import io.deephaven.parquet.table.plugin.channelprovider.tracked.TrackedSeekableChannelsProvider;
```

Comment:
Looks like this is why you needed the implementation dependency; and as far as I can tell, the only external caller for `TrackedSeekableChannelsProvider.getInstance()`. I think we should try and service this via `io.deephaven.parquet.base.util.SeekableChannelsProviderLoader#fromServiceLoader` instead, and then we could get rid of the direct dependency.